pkg/tiller/release_server_test.go

```diff
@@ -197,8 +197,9 @@ func TestValidName(t *testing.T) {
 		" ":                      false,
 		".nina.":                 false,
 		"nina.pinta":             true,
+		"abcdefghi-abcdefghi-abcdefghi-abcdefghi-abcdefghi-abcd": false,
 	} {
-		if valid != ValidName.MatchString(name) {
+		if valid != (validateReleaseName(name) == nil) {
```

Comment:
@adamreese  Many thanks for you to review the code. I modified and fixed the code as you mentioned. Please help me to review the code, thanks a billion.